server/liveblog/embed/embed_assets/themes/default-theme/main.js

```diff
@@ -40,12 +53,58 @@
             isAllowedToLoadMore: function() {
                 return !vm.loading && !vm.finished;
             },
+            getPermalink: function(id) {
+                var permalink = false,
+                    newHash = PERMALINK_HASH + '=' + id;
+
+                if (href.indexOf(PERMALINK_HASH_MARK) === -1) {
+                    permalink = href + PERMALINK_HASH_MARK + newHash;
+                } else if (href.indexOf(PERMALINK_HASH + '=') !== -1) {
+                    var regexHash = new RegExp(escapeRegExp(PERMALINK_HASH) + '=[^&#]*');
+                    permalink = href.replace(regexHash, newHash);
+                } else {
+                    permalink = href + '&' + newHash;
+                }
+                return permalink;
+            },
+            getPermalinkId: function() {
+                var matches, 
+                    regexHash = new RegExp(escapeRegExp(PERMALINK_HASH) + '=([^&#]*)');
+                vm.permalinkId = (matches = href.match(regexHash)) ? matches[1] : false;
+
+            },
+            scrollToPermalink: function() {
+                if(!vm.permalinkId) {
+                    return;
+                }
+                var posts = vm.pagesManager.allPosts(),
+                    found = false;
+                angular.forEach(posts, function(post) {
+                    if(post._id === vm.permalinkId) {
+                        found = true;
+                    }
+                });
+                if (!found && vm.isAllowedToLoadMore()) {
+                    vm.fetchNewPage().then(function() {
+                        vm.scrollToPermalink();
+                    });
+                } else if(found) {
+                    $timeout(function(){
+                        $anchorScroll(vm.permalinkId);                        
+                    });
+                }
+            },
             pagesManager: new PagesManager(POSTS_PER_PAGE, DEFAULT_ORDER)
         });
+
+        vm.getPermalinkId();
```

Comment:
A getter should return a value, and this value should be assigned when called, not inside the getter.